### PR TITLE
[SHOULD BE READY™][SEMI-MODULAR]Synthetic wounds

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
title

## Why It's Good For The Game
balance

## Changelog
:cl:
refactor: oh god oh fuck merge conflict time
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
Early to ask but I figured I might as well, will this come with making synths not ridiculously fragile any longer?

**Comment by KasparoVy** [comment]
> 
> 
> Early to ask but I figured I might as well, will this come with making synths not ridiculously fragile any longer?

I hope so but wouldn't hold out too much hope on it

**Comment by Vizoplez** [comment]
> Early to ask but I figured I might as well, will this come with making synths not ridiculously fragile any longer?

synths will no longer die to a snowflake

**Comment by KasparoVy** [comment]
GOD BLESS

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/burn.dm
Comment (ZephyrTFA) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/burn.dm]:
```suggestion
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/burn.dm
Comment (ZephyrTFA) [reply modular_skyrat/modules/syntheticwound/code/datums/wounds/burn.dm]: Why does this exist if you dont do anything with it and there is also no else statement

**Comment by Vizoplez** [comment]
PR is literally on-fucking-hold until Lummox sorts their shit out and allocates more memory to the compiler.

**Comment by Vizoplez** [comment]
time to manually fix errors by using CI as a method of code reviewing.

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/pierce.dm
Comment (ZephyrTFA) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/pierce.dm]:
```suggestion
	tape(I, user)
```

**Comment by Vizoplez** [comment]
ok we may actually have a compiling code here

**Comment by Vizoplez** [comment]
image

**Comment by Vizoplez** [comment]
currently runtimes when trying to create wounds, due to the list being fucky.

**Comment by Vizoplez** [comment]
ok its probably ready this has been barely tested please speedmerge maintainers

**Comment by Vizoplez** [comment]
this compiled fine, we just have monkey runtime 2.0 involving blood splatters.

**Comment by Vizoplez** [comment]
ok it seems to work

**Comment by Vizoplez** [comment]
Okay, everything actually works, lets FUCKING GOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by Gandalf2k15** [comment]
Thank the lord. As a synth main, this is a god send. Finally.

**Comment by SpaceVampire** [comment]
I really don't think that synths need cold resistance.
This wound thing is just trying to put em more on level with everyone else.
They still have a lot of major upsides left.
Oh also, leave the payday_modifier alone.
If you actually look, you would see that the other custom races.
Have a reduced one as well.
EDIT: You also lowered the heat vulnerability at the same time. It was like. Three before.

**Comment by hugmeorelse** [comment]
Synths must suffer too

**Comment by Vizoplez** [comment]
> I really don't think that synths need cold resistance.
> This wound thing is just trying to put em more on level with everyone else.
> They still have a lot of major upsides left.
> Oh also, leave the payday_modifier alone.
> If you actually look, you would see that the other custom races.
> Have a reduced one as well.
> EDIT: You also lowered the heat vulnerability at the same time. It was like. Three before.

you have absolutely no clue what this PR does to synths and I'm loving it

**Comment by SpaceVampire** [comment]
> you have absolutely no clue what this PR does to synths and I'm loving it

I mean. You made the comment in the code.
//We removed this. Stop touching payday modifiers.
Which is factually incorrect if you look at the other races like I said.
So really, it would have been nice to get an actual reply or explanation.
Because I did take the time to read over all of the code, to try and get a feel for it.
And you really didn't offer anything in your documentation above.

**Comment by Vizoplez** [comment]
> > you have absolutely no clue what this PR does to synths and I'm loving it
> 
> I mean. You made the comment in the code.
> //We removed this. Stop touching payday modifiers.
> Which is factually incorrect if you look at the other races like I said.
> So really, it would have been nice to get an actual reply or explanation.
> Because I did take the time to read over all of the code, to try and get a feel for it.
> And you really didn't offer anything in your documentation above.

yes its why its testmerged, it is still le WIP, and will be changed.

Wait until you see gameplay. Then you will regret.

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:124]:
```suggestion
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:127]:
```suggestion
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:105]: this can be shifted around with a ! but I'm too tired to figure it out rn

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:152]: I

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:158]: I

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:166]: maybe I'm an idiot but what is xadone

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:176]:
```suggestion
	var/self_penalty_multiplier = (user == victim ? 1.25 : 1)
```

don't really lose anything from writing it out fully here

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:190]: I

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:194]:
```suggestion
	user.visible_message("<span class='danger'>[user] begins welding over [user == victim ? "[user.p_their]" : "[victim]'s"] [limb.name] with [I]...</span>", "<span class='warning'>You begin welding over [user == victim ? "your" : "[victim]'s"] [limb.name] with [I]...</span>")
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:198]:
```suggestion
	user.visible_message("<span class='green'>[user] welds over some of the leaking on [user == victim ? "[user.p_their]" : "[victim]'s"] [limb.name].</span>", "<span class='green'>You weld over some of the leaking on [user == victim ? "[user.p_their]" : "[victim]'s"] [limb.name].</span>")
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:211]: B

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:230]:
```suggestion
	desc = "Patient's metallic coating has been sheared, causing internal leaking."
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:231]:
```suggestion
	treat_text = "Liberal application of surgical tape and welding"
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm:232]:
```suggestion
	examine_desc = "has a torn panel, oil leaking out slowly."
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm]:
```suggestion
	desc = "Patient's metallic outer coating has been entirely ripped open, exposing sensitive hydraulics."
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm]:
```suggestion
	treat_text = "Speedy application of surgical tape to stem the initial bleeding, followed by welding over the affected area"
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm]:
```suggestion
	examine_desc = "has severe damage to their limb, oil leaking heavily"
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm]:
```suggestion
	occur_text = "is ripped open, its panel sparking and its hydraulics leaking oil"
```

File: modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm
Comment (tf-4) [inline modular_skyrat/modules/syntheticwound/code/datums/wounds/slash.dm]:
```suggestion
	examine_desc = "has been absolutely destroyed, sparking electronics protruding from their limb"
```

**Review by tf-4 — COMMENTED** [review]
Mostly just variable naming and spelling/grammar.

File: code/datums/wounds/_wounds.dm
Comment (Vizoplez) [reply code/datums/wounds/_wounds.dm:116]: originally the wound flags were adjusted, remnants. Will be removed before final merge

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm
Comment (Vizoplez) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm:26]: it was only raised to 3 to nerf them in SEEK's PR

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm
Comment (Vizoplez) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm:28]: fair but this is also entirely deprecated as per Azarak's PR and should be set to 1 either way.

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm
Comment (Vizoplez) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm:42]: this was only moved, and was how the code was before. Not in the scope of the PR

**Comment by Vizoplez** [comment]
thank you :)

File: code/datums/wounds/_wounds.dm
Comment (Vizoplez) [reply code/datums/wounds/_wounds.dm:110]: ok thats fair but i'll wait until final merge :)

**Comment by Vizoplez** [comment]
No longer involved with this server. Feel free to mirror this branch and fix it up later.

**Comment by Gandalf2k15** [comment]
Noo, i gotta do this

**Comment by Vizoplez** [comment]
n o

**Pull request closed without merging**
